test/parallel/test-preload.js

```diff
@@ -123,7 +123,7 @@ const interactive = childProcess.exec(nodeBinary + ' '
   + '-i',
   common.mustCall(function(err, stdout, stderr) {
     assert.ifError(err);
-    assert.strictEqual(stdout, `> 'test'\n> `);
+    assert.strictEqual(stdout, '> \'test\'\n> ');
```

Comment:
I agree with @princejwesley or just leave it as is, escaped chars decrease readability imo.